contracts/scripts/set_config.sh

```diff
@@ -80,6 +87,15 @@ commands+=(
     "sozo execute --world "$world" SetLaborCostAmount --account-address $DOJO_ACCOUNT_ADDRESS --calldata 255,3,10"
 )
 
+
+commands+=(
+    # Define hyperstructure
+    # hyperstructure type 1
+    # resource 1,000 stone and 1,000 shekels
+    # hyperstructure coordinate x: 800, y : 200 
+    "sozo execute --world $world DefineHyperStructure --account-address $DOJO_ACCOUNT_ADDRESS --calldata 1, 2,2,1000,3,1000, 800,200"
```

Comment:
@aymericdelab Used arbitrary values. You can  reply with correct values if necessary